proposals/lip-0057.md

```diff
@@ -1351,6 +1351,7 @@ def getActiveValidators(validatorsTwoRoundsAgo: list[EligibleValidatorObject], r
         # Determine active validators and their BFT weight.
         activeValidators = [{"address": item.address, "bftWeight": ceiling(item.weight, WEIGHT_SCALE_FACTOR) } for item in validatorsTwoRoundsAgo[:nbrElectedValidators]]
         if len(activeValidators) > 0:
+            # average BFT weight can not be 0, since BFT weight of elected validators is at least 1. 
```

Comment:
I would explicitly mention that this is because `MIN_WEIGHT>=1`

Reply:
Done